pages/saved.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Image from 'next/image'
+import { FaArrowRight } from "react-icons/fa";
+import Link from 'next/link';
+
+const Saved = () => {
+  return (
+    <>
+      <div className="p-8">
+        <h1 className='text-2xl font-medium'>Saved</h1>
+        <p className='text-[#A6ABBF]'>Your saved resources, ready when you are.</p>
+      </div>
+      <div className='flex h-screen justify-center items-center flex-col'>
```

Comment:
We don't need that much space. This will work. 

```suggestion
      <div className='flex min-h-[calc(100vh-256px)] justify-center items-center flex-col'>
```